index.html

```diff
@@ -0,0 +1,186 @@
+         taking place in October, in Rwanda.
+            </p>
+            <div class="date">2023.10.15(THU) ~ 16(FRI)</div>
+            <div class="location">
+              @ Kigali Convention Center
+            </div>
+          </section>
+    </div>
+    
+    <!-- Main program section -->
+    <section class="section-2" id="program">
+        <div class="h2-red-line">
+            <h2 class="h2">Main Program</h2>
+            <hr class="red-line">
+        </div>
+
+        <div class="container">
+            <!-- Box 1 -->
+            <div class="inner-container">
+                <img src="images/program_icon_01.png" alt="image" class="icon">
+                <h3 class="h3">Lecture</h3>
+                <p class="p">
+                    Listen to the speakers from countries about the messages of sharing
+                    and opening.
+                  </p>
+              </div>
+
+                  <!-- Box 2 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_02.png" alt="image" />
+                    <h3 class="h3">TMS Exibition</h3>
+                    <p class="p">
+                      Appreciate various creations applying TMS license of artists,
+                      organized from Art Center Nabi.
+                    </p>
+                  </div>
+        
+                  <!-- Box 3 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_03.png" alt="image" />
+                    <h3 class="h3">Forum</h3>
+                    <p class="p">
+                      Have the time to share your thoughts and opinions with experts for
+                      each topic.
+                    </p>
+                  </div>
+        
+                  <!-- Box 4 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_04.png" alt="image" />
+                    <h3 class="h3">Hands-On Workshops:</h3>
+                    <p class="p">
+                      Try creating your own work using open source license rather than
+                      just watching at it.
+                    </p>
+                  </div>
+        
+                  <!-- Box 5 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_05.png" alt="image" />
+                    <h3 class="h3">Networking Opportunities</h3>
+                    <p class="p">
+                      Get opportunities to network with TMS affiliates around the world,
+                      also after the summit.
+                    </p>
+                  </div>
+                </div>
+        
+                <div class="red-box">Join  TechMingle Summit 2023</div>
+                <div class="whole-program">SEE THE WHOLE PROGRAM</div>
+              </section>
```

Comment:
- [ ] [Optional] This is small changes so I am giving it optional, Good job on making the changes please make sure that give the container certain height so It looks great.
### Your design
![his](https://github.com/gilberthappi/TechMingle_Summit/assets/81727988/0fc85eb9-c4f0-4a13-857d-71118da7db31)
### After I give certain height to container
![sa](https://github.com/gilberthappi/TechMingle_Summit/assets/81727988/df016b25-a237-4842-9d92-33e9640ca744)

Reply:
Okey Thanks